CMakeLists.txt

```diff
@@ -1,27 +1,40 @@
-cmake_minimum_required(VERSION 2.8.3)
+cmake_minimum_required(VERSION 3.5)
 project(pcl_conversions)
 
-find_package(catkin REQUIRED COMPONENTS pcl_msgs roscpp sensor_msgs std_msgs cmake_modules)
+if(NOT WIN32)
+  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11 -Wall -Wextra")
+endif()
+
+find_package(ament_cmake REQUIRED)
+find_package(ament_cmake_gtest REQUIRED)
 
+find_package(Boost REQUIRED)
+find_package(builtin_interfaces REQUIRED)
+find_package(Eigen3 REQUIRED)
+# TODO(clalancette): Re-enable this once we have pcl_msgs
+#find_package(pcl_msgs REQUIRED)
 find_package(PCL REQUIRED QUIET COMPONENTS common)
```

Comment:
Quiet just means it will suppress extra console output, it does not prevent it from failing. Required just makes it fail if it is not found. These options are sort of orthogonal.